music_assistant/controllers/player_queues.py

```diff
@@ -1670,6 +1670,7 @@ async def _get_radio_tracks(
     ) -> list[Track]:
         """Call the registered music providers for dynamic tracks."""
         queue = self._queues[queue_id]
+        queue_track_items = [q.media_item for q in self._queue_items[queue_id]]
```

Comment:
```suggestion
        queue_track_items = [q.media_item for q in self._queue_items[queue_id] if q.media_item]
```
The queue can (theoreticaly) also contain queueitems without a mediaitem attached, lets guard for that